tests/components/conversation/__init__.py

```diff
@@ -44,6 +48,53 @@ async def async_process(self, user_input: ConversationInput) -> ConversationResu
         )
 
 
+@pytest.fixture
+async def mock_chat_log(hass: HomeAssistant) -> conversation.MockChatLog:
```

Comment:
Is this intentionally not just `MockChatLog`? I notice you have the noqa when consuming this so wondering why this type

Reply:
Oh that’s a bug

Reply:
```suggestion
async def mock_chat_log(hass: HomeAssistant) -> MockChatLog:
```